test/e2e/provisioning/testdata/machinedeployment-hetzner.yaml

```diff
@@ -30,6 +30,8 @@ spec:
             location: "nbg1"
             networks:
               - "machine-controller-e2e"
+            firewall:
+              - "machine-controller-e2e"
```

Comment:
This might fail our tests because I don't think we have created this firewall yet.